src/Behat/Config/Config.php

```diff
@@ -4,13 +4,27 @@
 
 namespace Behat\Config;
 
+use function is_string;
+
 final class Config implements ConfigInterface
 {
     public function __construct(
         private array $settings = []
     ) {
     }
 
+    /** @param string|string[] $resource **/
+    public function import(string|array $resource): self
+    {
+        $resources = is_string($resource) ? [$resource] : $resource;
```

Comment:
I would personally use a variadic here, it gives us (and users) strict typing out of the box including protecting against unexpected/unused extra arguments, keeps our own code as thin as possible, and therefore reduces any risk of edge cases.

As a user, the syntax `->import('one.php', 'two.php')` is also cleaner and easier to format IMO than having the redundant array braces.

I know there are other tools out there that do `string|string[]` but my sense is those are interfaces that were designed before variadics were available and so they're tied to them. I don't think we should be, unless there's a reason not to use a variadic that I haven't thought of?

Reply:
Fair enough, makes sense.